tools/generate_games_list.js

```diff
@@ -22,11 +22,12 @@ sortedGamesIds.forEach(key => {
 
 let generated = ''
 generated += '| GameDig Type ID | Name | See Also\n'
-generated += '|---|---|---\n'
```

Comment:
Could you revert this file just as it was?

Reply:
(also the games_list file because of this)

Reply:
Yeah, of course!